### DISCUSSION TITLE
Notifiche periodo validità (campo dataFineValidita)

### DISCUSSION THREAD

**Discussion by migmar68** [italia/ansc#1161] [Supporto servizi cooperativi]
Buongiorno 
Nella risposta alle notifiche, sono previsti questi due campi (riporto un esempio dei valori restituitiì) : 

```
			"dataInizioValidita": "2024-10-02T16:23:30Z",
			"dataFineValidita": "9999-12-31T16:23:30Z",
```

Volevo sapere se la data di fine è inserita a possibile uso futuro, o se abbiamo già dei casi pratici dove viene applicata una data non infinita, e quindi dobbiamo interpretare la risposta, e in che modo.

Marco Migliori

**Comment by asqueglia** [comment]
Buon pomeriggio, attualmente non è gestita la data di fine validità delle notifiche e al momento non è in previsione alcuno sviluppo in merito. Si chiude pertanto la discussion
